Python/modsupport.c

```diff
@@ -33,6 +33,19 @@ _Py_convert_optional_to_ssize_t(PyObject *obj, void *result)
     return 1;
 }
 
+int
+_Py_convert_optional_to_non_negative_ssize_t(PyObject *obj, void *result)
+{
+    if (!_Py_convert_optional_to_ssize_t(obj, result)) {
+        return 0;
+    }
+    if (obj != Py_None && *((Py_ssize_t *)result) < 0) {
+       	PyErr_SetString(PyExc_ValueError, "argument must be at least 0");
+    	return 0;
```

Comment:
mixed tabs and spaces. @wiomoc can you check elsewhere in the PR that no tabs have snuck in?
```suggestion
        return 0;
```